id-repository/id-repository-credentials-feeder/pom.xml

```diff
@@ -10,7 +10,7 @@
 	<parent>
 		<groupId>io.mosip.idrepository</groupId>
 		<artifactId>id-repository-parent</artifactId>
-		<version>1.2.1-SNAPSHOT</version>
+		<version>1.2.0.1</version>
```

Comment:
why idrepo-parent version is 1.2.0.1

Reply:
Credential feeder is not migrated so using idrepo parent 1201

Reply:
[MOSIP-35146](https://mosip.atlassian.net/browse/MOSIP-35146) 
https://mosip.atlassian.net/browse/MOSIP-35147
jira tickets attatched.